tests/qc/test_roi_qc.py

```diff
@@ -1,11 +1,11 @@
 from typing import Union
 import pytest
-
-import numpy as np
-from scipy.sparse import coo_matrix
-import matplotlib
-
-from ophys_etl.qc import roi_qc
+import matplotlib as mpl
+mpl.use('Agg')
+import matplotlib  # noqa: E402
+import numpy as np  # noqa: E402
```

Comment:
Same as above, for sure you can move scipy and numpy up to get rid of flake8. Not sure if ophys_etl.qc imports matplotlib, but if it doesn't you can move that too.

Reply:
done.